helix-core/src/main/java/org/apache/helix/manager/zk/zookeeper/ZkConnection.java

```diff
@@ -84,6 +83,25 @@ public void close() throws InterruptedException {
     }
   }
 
+  protected void reconnect(Watcher watcher) throws InterruptedException {
+    _zookeeperLock.lock();
+    try {
+      if (_zk == null) {
+        throw new IllegalStateException("zk client has not been connected or already been closed");
+      }
+      ZooKeeper prevZk = _zk;
+      try {
+        LOG.debug("Creating new ZookKeeper instance to reconnect to " + _servers + ".");
```

Comment:
LOG.info?

Reply:
We already have state change logs. In addition, if the reconnect fails, we will print warning log in the ZkClient. Here, to be consistent with other ZkConnection, I think it's better that we keep it a debug log.